packages/babel-parser/test/fixtures/comments/basic/async-do-expression/output.json

```diff
@@ -0,0 +1,90 @@
+{
+  "type": "File",
+  "start":0,"end":44,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":44}},
+  "program": {
+    "type": "Program",
+    "start":0,"end":44,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":44}},
+    "sourceType": "script",
+    "interpreter": null,
+    "body": [
+      {
+        "type": "ExpressionStatement",
+        "start":7,"end":37,"loc":{"start":{"line":1,"column":7},"end":{"line":1,"column":37}},
+        "leadingComments": [
+          {
+            "type": "CommentBlock",
+            "value": " 1 ",
+            "start":0,"end":7,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":7}}
+          }
+        ],
+        "trailingComments": [
+          {
+            "type": "CommentBlock",
+            "value": " 5 ",
+            "start":37,"end":44,"loc":{"start":{"line":1,"column":37},"end":{"line":1,"column":44}}
+          }
+        ],
+        "expression": {
+          "type": "DoExpression",
+          "start":19,"end":37,"loc":{"start":{"line":1,"column":19},"end":{"line":1,"column":37}},
+          "leadingComments": [
```

Comment:
This is an unrelated bug: An async do expression should start at the position of `async`, here it starts at `do`. I will fix this issue in separate PR and rebase this one on top of that.